src/com/google/common/css/compiler/gssfunctions/GssFunctions.java

```diff
@@ -459,6 +467,454 @@ public Color addValuesToHsbComponents(Color baseColor,
   }
 
   /**
+   * Base implementation of color manipulation functions in HSL color space
+   */
+  public abstract static class BaseHslColorManipulation {
+
+    protected String addHslToCssColor(
+      String baseColorString, String hueToAdd, String saturationToAdd,
+      String lightnessToAdd) throws GssFunctionException {
+      try {
+        return addHslToCssColor(
+          baseColorString,
+          Integer.parseInt(hueToAdd),
+          Integer.parseInt(saturationToAdd),
+          Integer.parseInt(lightnessToAdd));
+      } catch (NumberFormatException e) {
+        String message = String.format("Could not parse the integer arguments"
+            + " for the function 'addHslToCssColor'. The list of arguments was:"
+            + " %s, %s, %s, %s. ",
+            baseColorString, hueToAdd, saturationToAdd, lightnessToAdd);
+        throw new GssFunctionException(message);
+      } catch (IllegalArgumentException e) {
```

Comment:
If you want to use Java 7's multi-catch to merge the two catch blocks, I'm OK with that.